simulator/property_collector.go

```diff
@@ -285,7 +285,7 @@ func (rr *retrieveResult) collectFields(ctx *Context, rval reflect.Value, fields
 
 		switch err {
 		case errEmptyField:
-			// ok
+			*emptyField = true
```

Comment:
See comment below, maybe we should append the empty property to PropSet here instead?